### DISCUSSION TITLE
migrating from v1

### DISCUSSION THREAD

**Discussion by jamesholcomb** [jpudysz/react-native-unistyles#120] [Q&A] [answered]
The upgrade doc was helpful!  The only two things I ran into (being a 2.0 noob) that could be considered doc additions...

- You need to `npx pod-install`
- You will get `warning " > react-native-unistyles@2.0.0" has unmet peer dependency "@react-native/normalize-colors@*".` 

I was able to build and run without it, but not sure if the second item is required?  If so, should it be a dependency or devDependency?

Great job on 2.0 🎉

**Comment by jpudysz** [comment, marked as answer]
Hello 👋 

Feel free to ignore this warning. I am utilizing some React Native internals within Unistyles. These will be entirely removed in version 3.0.

Regarding the ‘pod install’, it is mentioned only here: https://reactnativeunistyles.vercel.app/start/setup/. I believe adding it to migration guide would be beneficial. Thank you for the suggestion 🙏

**Comment by jpudysz** [comment]
Done: https://github.com/jpudysz/react-native-unistyles/pull/124
